XouDevSpec/Classes/Model/ApiCallerModel.swift

```diff
@@ -28,10 +28,11 @@ public struct ApiCallerModel {
     
     
     public  func getAnimeData(completetionHandler: @escaping(Result<[AnimeDetails], AnimeError>) -> Void) {
+    
+        var request = URLRequest(url: NSURL(string:  "https://jikan1.p.rapidapi.com/search/anime?q=\(searchText)")! as URL,
```

Comment:
Try to avoid force unwrapping and instead have a guard

Reply:
noted = )